src/main.ts

```diff
@@ -12,6 +12,8 @@ export const options: ts.CompilerOptions = {
 
 export class Minifier {
   static reservedJSKeywords = Minifier.buildReservedKeywordsMap();
+  private renameMap: {[name: string]: string} = {};
```

Comment:
Could you add comments as to what exactly the keys and values are (e.g. theoretically the keys could be fully qualified names + properties or something like that)?